File: scripts/pull_latest.py
import subprocess
import os

os.chdir('/vercel/share/v0-project')

try:
    result = subprocess.run(['git', 'pull', 'origin', 'main'], capture_output=True, text=True)
    print(result.stdout)
    if result.stderr:
        print("Stderr:", result.stderr)
    print("Pull completed successfully!")
except Exception as e:
    print(f"Error: {e}")

File: scripts/pull-latest.mjs
import { execSync } from 'child_process';
import { chdir } from 'process';

try {
  chdir('/vercel/share/v0-project');
  const output = execSync('git pull origin main', { encoding: 'utf-8' });
  console.log(output);
} catch (error) {
  console.error('Error pulling latest:', error.message);
}
